plugins/performance-lab/readme.txt

```diff
@@ -71,6 +71,14 @@ Contributions are always welcome! Learn more about how to get involved in the [C
 
 == Changelog ==
 
+= 3.8.0 =
+
+**Enhancements**
+
+* Add Site Health check for `Cache-Control: no-store` page response header which disables bfcache. ([1807](https://github.com/WordPress/performance/pull/1807))
+* Add Site Health test to verify that static assets are served with far-future expires. ([1727](https://github.com/WordPress/performance/pull/1727))
```

Comment:
Nit-pick: Seeing one changelog entry with `check` and another with `test` makes me think this should probably be aligned to use one or the other.